docs/apis/table.md

```diff
@@ -873,7 +920,7 @@ val result = tableEnv.sql(
 
 #### Limitations
 
-The current version of streaming SQL only supports `SELECT`, `FROM`, `WHERE`, and `UNION` clauses. Aggregations or joins are not supported yet.
+The current version of streaming SQL only supports `SELECT`, `FROM`, `WHERE` and `UNION` clauses. Aggregations or joins are not supported yet.
```

Comment:
We try to consistently use [serial commas](https://en.wikipedia.org/wiki/Serial_comma) in this document.